node/cli/src/chain_spec.rs

```diff
@@ -336,7 +336,10 @@ fn local_testnet_genesis() -> GenesisConfig {
 			get_authority_keys_from_seed("Bob"),
 		],
 		get_account_id_from_seed("Alice").into(),
-		None,
+		Some(vec![
```

Comment:
Final parameter is a `Vec<AccountId>`, you're passing in a coerced `Vec<AuthorityId>`. There is no reason to believe they will continue to be in any way compatible. Right now you can do such stuff because it's all v ery weakly-typed but as of https://github.com/paritytech/substrate/pull/1795 this code will (quite rightly) break.

Reply:
When will this pr be merged? Is there a better way of doing it currently?

The problem is that I need to have the stash accounts to have the balance.